src/utilities.jl

```diff
@@ -128,6 +128,8 @@ function convert_unit(value::T, unit1::Symbol, unit2::Symbol) where {T<:Real}
     return _FType(T)(value * (_energy_units[unit1] / _energy_units[unit2]))
 end
 
+get_typename_wrapper(A) = Base.typename(typeof(A)).wrapper
```

Comment:
Why should we support this for arbitrary input?

Reply:
for the function
`_lazy_tensor_warning`